mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -2329,12 +2347,26 @@ impl UtxoRpcClientOps for ElectrumClient {
             rpc_req!(self, "blockchain.scripthash.get_balance").into(),
             JsonRpcErrorType::Internal(err.to_string())
         )));
-        let hash = electrum_script_hash(&output_script);
-        let hash_str = hex::encode(hash);
-        Box::new(
-            self.scripthash_get_balance(&hash_str)
-                .map(move |electrum_balance| electrum_balance.to_big_decimal(decimals)),
-        )
+        let mut hashes = vec![hex::encode(electrum_script_hash(&output_script))];
+
+        // If the plain pubkey is available, fetch the balance found in P2PK output as well (if any).
+        if let Some(pubkey) = address.pubkey() {
```

Comment:
should we not add the same code into fn display_balances() too?

Reply:
as we agreed HD accounts do not support P2PK so we do not need this in display_balances